app/src/main/AndroidManifest.xml

```diff
@@ -61,9 +61,14 @@
             </intent-filter>
 
             <intent-filter>
-                <action android:name="android.intent.action.SEND" />
```

Comment:
Adding the new intent makes sense to me. But we do not want to remove the SEND intent :)

Reply:
Sorry, that wasn't intentional!